build-system/common/browserify-cache.js

```diff
@@ -59,3 +67,11 @@ function getPersistentBrowserifyCache() {
 module.exports = {
   getPersistentBrowserifyCache,
 };
+
+/**
+ * Prints the object-hash of the hashObject representing all the files we care
+ * about. Used during CI to determine when to refresh .karma-cache.
+ */
+if (require.main === module) {
```

Comment:
This isn't forward looking (it won't work in ESM). I would suggest creating a purpose-built file (with `#!/usr/bin/env node` hashbang) to log this.